### DISCUSSION TITLE
several questions on selfhosting

### DISCUSSION THREAD

**Discussion by jamiew0w** [stoatchat/discussions#284] [Q&A] [answered]
hello guys and girls.

i've somewhat lazily followed revolt for a while and i think it's time i stood up my own instance so i can test and involve myself a bit more but before i do so:

1. are there any plans to incorporate mysql instead of mongodb?
2. is "docker only" just for development or permanent? obviously it's far easier to setup but i much prefer working with individual services (yes i know how dumb it sounds).

my apologies for the mundane questions and thanks for everything you're doing.

**Comment by Zomatree** [comment, marked as answer]
1. Yes (ish), we are working on a library to abstract over multiple databases so people could use other databases, mysql could be added but its not high priority.
2. Nothing is "docker only", you are able to build and run everything without docker.

**Reply by JoeJoeTV** [reply]
To 2. if this doesn't sound to dumb: Will the docker option be the only one supported or will there be atleast some documentation to set up revolt without docker?

**Reply by Ikun886-cxk** [reply]
Yes, at present, there is only a tutorial on Docker construction.
